SkyEditor.RomEditor.Rtdx.ConsoleApp/Scripts/CSharp/SkyEditor.csx

```diff
@@ -8,4 +8,7 @@
 // If you're editing the template from the release
 #r "../../SkyEditor.RomEditor.Rtdx.dll"
 
+using System;
```

Comment:
Last I tried, these didn't apply to whatever script references this. Does it work differently for you?

Reply:
Yup, I used VS Code and didn't test it in regular VS though.

Reply:
Actually, I guess this is undesired behaviour because it would leak imports into other files. Maybe it's a bug in the VS Code extension and not supposed to work so we should get rid of it for clarity.

Reply:
I would personally want it to work. See [here](https://github.com/evandixon/SkyEditor.RomEditor.Rtdx/blob/master/SkyEditor.RomEditor.Rtdx.ConsoleApp/Scripts/CSharp/Template.csx#L13) for why.